client/app/src/actions/bookAction.js

```diff
@@ -0,0 +1,208 @@
+import axios from 'axios';
+import {
+  GET_ALL_BOOKS,
+  GET_ALL_BOOKS_ERROR,
+  BORROW_A_BOOK,
+  BORROW_A_BOOK_ERROR,
+  GET_USER_HISTORY,
+  GET_USER_HISTORY_ERROR,
+  RETURN_A_BOOK,
+  RETURN_A_BOOK_ERROR,
+  ADD_BOOK,
+  ADD_BOOK_ERROR,
+  DELETE_BOOK,
+  DELETE_BOOK_ERROR
+} from '../constants/actionTypes';
+
+/**
+ *
+ * @param {msg} message - dispatched message object
+ */
+const deleteBook = deleteMessage => ({
+  type: DELETE_BOOK,
+  deleteMessage
+});
+/**
+ *
```

Comment:
nice use of jsdoc. you can also go further by adding a concise description of what your functions do.  Also, it is considered a good practice to specify what your method is returning in your jsdoc. Please visit [this page](http://usejsdoc.org/tags-description.html) to see an example of a minimal and good jsdoc